### ISSUE TITLE
Bnspatial package compatible with Rversion 4.33

### ISSUE THREAD

**Issue by CMzyece** [dariomasante/bnspatial#15]
Goodmorning, is there bnspatial package compatible with R version 4.33? Its currently not available in the library for this R version but am getting a message saying it maybe available elsewhere. How can I install it?

**Comment by dariomasante** [comment]
Hi, I haven't updated bnspatial for a while now, so I dont know if it's compatible with 4.33, but for sure it is not compatible with some of the required packages (i.e. raster was discontinued and gRain changed a lot). 
I will update everything and put it back on CRAN at some point, but for the moment I can't provide you any solutions, I'm afraid

**Comment by CMzyece** [comment]
Thank you for getting back to me. I may just have to use R 3.6. Please keep me posted once the package is updated.

**Comment by CMzyece** [comment]
Hi Dario, so can bnspatial run at all? I thought getting a lower R version (3.6.3) would work but I see there's lots of error messages popping up. Are there any alternative packages you would recommend?

**Comment by dariomasante** [comment]
Yes as I mentioned in my previous message, bnspatial still depends on raster packages (now discontinued) and an older gRain version (changed a lot), that's why bnspatial crashes and was removed from cran, because it needs to be updated.
Try have a look at this alternative https://www.sciencedirect.com/science/article/pii/S1364815219306061

**Comment by CMzyece** [comment]
Hi Dario, thanks for clarifying and for the alternative too. That's really helpful.
